### DISCUSSION TITLE
Run on a Ubuntu VM

### DISCUSSION THREAD

**Discussion by LiamBMX** [KevinBytesTheDust/CVAmp#259] [Q&A]
Yo I have an Ubuntu VM, could I run the file on that? Idk much abt vm's so idk if It would be possible with just ssh and sftp access

Also please add lower bandwidth usage, I bought 100 proxies and I used 8/250gb of bandwidth in one stream

**Comment by jlplenio** [comment]
The tool automatically chooses the lowest resolution possible. Please check beforehand if the stream offers lower resolutions than source.

Natively, no. But if you have to have some Python experience (step5 is not easy), you could:

1. Install Python version 3.11.X
2. Download the repo, go into it and execute the commands:
3. pip install poetry
4. poetry install
5. Use the information from this old version to start the manager in REPL and add instances: 
https://github.com/jlplenio/crude-twitch-viewer-bot/blob/v0.1.2/main.py
